.github/workflows/gitleaks.yml

```
@@ -1,50 +1,6 @@
-    name: gitleaks full-history scan
-    runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      pull-requests: write
-    steps:
-      - name: Checkout full history
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-
       - name: Run gitleaks (action)
```

actionlint (1.7.9)

"jobs" section is missing in workflow

(syntax-check)